### PR TITLE
chore(deps): Update dependencies to latest versions.

### PR DESCRIPTION
**Pull request by rfk**
@vladikoff r?

### CODE REVIEW

**Comment by rfk** [comment]
Hmm, test fails on node 0.10, are we in a position to drop 0.10 from this repo yet?

**Comment by vladikoff** [comment]
> are we in a position to drop 0.10 from this repo yet?

hm i think not yet, but soon!

**Comment by rfk** [comment]
I downgraded some grunt-related deps that seemed to cause trouble on node 0.10, let's see how this one goes...

File: package.json [hunk 1]
```diff
@@ -35,15 +35,16 @@
...
     "grunt-copyright": "0.3.0",
-    "grunt-eslint": "19.0.0",
+    "grunt-eslint": "17.3.1",
     "grunt-jscs": "3.0.1",
     "grunt-mocha-test": "0.13.2",
     "grunt-nodemon": "0.4.2",
     "grunt-nsp": "2.3.1",
     "grunt-todo": "0.5.0",
     "jscs-jsdoc": "2.0.0",
+    "mocha": "^3.1.0",
```
Comment (rfk) [inline package.json]: I think it somehow moved to a peer-dependency with one of the updates; in any case, tests refused to run without me explicitly installing it.

Comment (vladikoff) [inline package.json]: Thought so, we can keep it! but should we lock it to a version?

Comment (rfk) [inline package.json]: Yep, updated to a specific version.

**Comment by rfk** [comment]
@vladikoff I don't know what to make of this circleci failure though: https://circleci.com/gh/mozilla/fxa-basket-proxy/8?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by vladikoff** [comment]
`peerinvalid Peer grunt-requirejs@0.4.2 wants grunt@~0.4.0` need to switch to https://github.com/mozilla/fxa-content-server/blob/master/package.json#L58

ref: https://github.com/asciidisco/grunt-requirejs/pull/112

**Comment by rfk** [comment]
Ah, thanks, I missed the little `~` there.  Although I deliberately removed `grunt-requirejs` because it doesn't seem to be needed, and is not referenced anywhere in my npm-shrinkwrap, so not entirely sure why Circle's trying to install it...

**Comment by vladikoff** [comment]
> so not entirely sure why Circle's trying to install it...

Ow, maybe it is cached, clearing cache..

**Comment by vladikoff** [comment]
> Loading "Gruntfile.js" tasks...ERROR
>> Error: env: must be one of the possible values: ["production","development"]: value was "test"
Warning: Task "test" not found. Use --force to continue.

🚥

**Comment by rfk** [comment]
Green!

**Comment by vladikoff** [comment]
it's real!

**Pull request merged**